### DISCUSSION TITLE
Can we get two tables of data with the blitz paginate query(findmany)? without using of rawquery and include.

### DISCUSSION THREAD

**Discussion by KhushbuAsrani08** [prisma/prisma#18297] [Q&A]
Can we get two tables of data with the blitz paginate query(findmany)? without using of rawquery and include.

**Comment by ludralph** [comment]
Hi @KhushbuAsrani08 👋 

Can you please elaborate on your question to enable me to help you further? An example might help as well.

**Reply by KhushbuAsrani08** [reply]
Hi @ludralph , Thanks for the reply....

```ts
 const {
          items: users,
          hasMore,
          nextPage,
          count,
        } = await paginate({
          skip,
          take,
          count: () =>
            resultdata.push(
              db.projectUser.count({
              })
            ),
          query: (paginateArgs) =>
            db.$transaction([
              db.projectUser?.findMany({
                ...paginateArgs,
              }),
              db.projectInvitationPendingUser.findMany({
                ...paginateArgs,
                orderBy,
              }),
            ]),
        })
```
 In the above query using transaction i am getting data of two different table. But in this case server side pagination is not possible. So is there any any by which i can get data of two table once inside paginate().

**Reply by ludralph** [reply]
Hi @KhushbuAsrani08 

Assuming there is a relation between the two tables `projectUser` and `projectInvitationPendingUser`, you can modify the `query` function inside the `paginate` function to perform a join query that fetches data from both tables. Here's an example:

```js
query: (paginateArgs) =>
  db.projectUser.findMany({
    ...paginateArgs,
    orderBy,
    take,
    skip,
    include: {
      projectInvitationPendingUser: true
    }
  })
```
In this example, we use Prisma's include parameter to perform a left join between projectUser and projectInvitationPendingUser tables, based on their common field. This will fetch data from both tables in a single query, and you can then apply pagination on the combined results.

**Reply by KhushbuAsrani08** [reply]
Hi @ludralph , Yes about this I am aware.There is no relation between two table and without raw query I want to implement it.
